.github/workflows/ci.yml

```diff
@@ -1580,7 +1578,6 @@ jobs:
         else
           emcmake cmake -DCMAKE_BUILD_TYPE=${{ env.BUILD_TYPE }}    \
                 -DUSE_CLING=OFF                             \
-                -DUSE_REPL=ON                               \
```

Comment:
please make the similar changes in this file too 

https://github.com/compiler-research/CppInterOp/blob/main/.github/workflows/deploy-pages.yml

Reply:
I suggest he goes through all workflows in the repo.

Reply:
Thanks for letting me know.